big-int/src/operators/not_equal_to.cpp

```diff
@@ -32,5 +32,20 @@
 
 namespace libbig {
 
+  bool largeInt:: operator != (const largeInt& z) {
+    if(this->sign != z.sign) {
+      return true;
+    }
+    if (this->number.length() != z.number.length()) {
+      return true;
+    }
+    for(int i = 0; i < this->number.length(); i++) {
+      if (this->number[i] != z.number[i]) {
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
Why does this have to be so complex? Operator == is already overloaded.

bool largeInt:: operator != (const largeInt z) {
  return !(*this == z);
}

This much code should be fine. Plus it will always be in sync with == operator.

Reply:
actually this PR was made before the overloading of `==` operator.
i was thiking to request changes on this
thanks @ufrshubham for pointing out

Reply:
Ahh, that makes sense.